src/internal/NeoMethods.h

```diff
@@ -37,6 +37,10 @@ License along with NeoPixel.  If not, see
 #include "methods/Sm16716GenericMethod.h"
 #include "methods/Mbi6033GenericMethod.h"
 
+//Adafruit Pixie via UART, not platform specific
```

Comment:
minor overall comment about Serial/Uart naming versus Stream naming.
The method really doesn't care what kind of stream it is, as you decided to place serial initiation/config in the hands of the sketch author.  So why then is it named "Serial" rather than "Stream"?

Reply:
Good suggestion, made the change.